src/main.cpp

```diff
@@ -55,17 +59,22 @@ int main(int argc, char *argv[]){
 
         if(gpu_flag){
             std::cout << "Running analysis using GPU" << std::endl;
-            auto program = parse_to_cfg_transformer<std::set<std::string>>(prog);
-            gpu_analysis(program.nodes);
-            print_digraph_subgraph(program.entryNodes, std::cout, print_result, "main");
+
+            auto program = timeFunc<ScTransformer<std::set<std::string>>>("Creating CFG nodes: ", 
+                parse_to_cfg_transformer<std::set<std::string>>, prog);
+            
+            timeFunc("Total GPU analysis: ", gpu_analysis, program.nodes);
+            // print_digraph_subgraph(program.entryNodes, std::cout, print_result, "main");
         }else{
             if(multi_taint_flag){
                 std::cout << "Running multi-taint analysis using CPU" << std::endl;
                 auto program = parse_to_cfg_transformer<SourcedTaintState>(prog);
-                cpu_multi_taint_analysis(program);
+                // cpu_multi_taint_analysis(program);
```

Comment:
Multi taint analysis probably shouldnt be disabled like this.
I would probably uncomment but just not apply any benchmarking to it